conf/supportability.yaml

```diff
@@ -1,4 +1,4 @@
 supportability:
   content_hosts:
     rhel:
-      versions: [6, 7, 8, 9]
+      versions: [6, 7,'7_fips', 8, '8_fips', 9]
```

Comment:
no `9_fips`?

Reply:
not yet, requres updates in deploy-base-rhel-fips, but soon